src/google-ads-api.ts

```diff
@@ -244,7 +262,44 @@ export class GoogleAdsApi {
           AND ad_group_ad.status != REMOVED
         `,
       FEED_ITEMS: `
-        SELECT extension_feed_item.image_feed_item.image_asset, extension_feed_item.resource_name, extension_feed_item.status, metrics.ctr, metrics.impressions FROM extension_feed_item WHERE extension_feed_item.extension_type = 'IMAGE' AND extension_feed_item.status != 'REMOVED'
+        SELECT 
+          extension_feed_item.image_feed_item.image_asset, 
+          extension_feed_item.resource_name, 
+          extension_feed_item.status, 
+          extension_feed_item.targeted_campaign,
+          extension_feed_item.targeted_ad_group,
+          metrics.ctr, 
+          metrics.impressions 
+        FROM 
+          extension_feed_item 
+        WHERE 
+          extension_feed_item.extension_type = 'IMAGE' 
+          AND extension_feed_item.status != 'REMOVED'
+      `,
+      AD_GROUP_ASSETS_FOR_CAMPAIGN_ID: `
+        SELECT
+          ad_group_asset.resource_name,
+          campaign.id
+        FROM 
+          ad_group_asset
+        WHERE
+          ad_group_asset.field_type = 'AD_IMAGE'
```

Comment:
Do we also need a status not removed filter here?

```suggestion
          ad_group_asset.field_type = 'AD_IMAGE'
          ad_group_asset.primary_status != 'REMOVED'
```